ui-grid/ui-grid.d.ts

```diff
@@ -3844,7 +3844,7 @@ declare module uiGrid {
          * or you can supply a custom filter function that gets passed the
          * following arguments: [searchTerm, cellValue, row, column].
          */
-        condition?: number;
+        condition?: number | ((searchTerm:string, cellValue:any, row:IGridRow, column:IGridColumn) => boolean);
```

Comment:
I'd consider separating the function into its own interface, but that's not necessary.... also you have extraneous parens (remove) and you're missing the space after the colon in the types:

```
condition?: number | (searchTerm: string, cellValue: any, row: IGridRow, column: IGridColumn) => boolean;
```